src/v3/src/util/resetMessagesToInputs.ts

```diff
@@ -14,7 +14,7 @@ import { IdxMessage, Input } from '@okta/okta-auth-js';
 
 export function resetMessagesToInputs(
   inputs: Input[],
-  messages: Record<string, Partial<IdxMessage>>,
+  messagesByField: Record<string, Partial<IdxMessage>[]>,
```

Comment:
renamed due to naming collision?

Reply:
Yes and no, this prop looks like this 
```
{ 
   "identifier": [{ i18n: { key: 'some.key'}}],
   "credentials.passcode": [{ i18n: { key: 'some.key'}}]
}
```
The name of the prop being "messages" isnt exactly reflective of what it stores. This is one reason why I renamed it, the second reason is because we actually grab the messages from the record below, and I wanted to use that variable name.